deps.bzl

```diff
@@ -2559,9 +2559,8 @@ def install_buildbuddy_dependencies(workspace_name = "buildbuddy"):
         name = "com_github_lni_dragonboat_v3",
         build_file_proto_mode = "disable_global",
         importpath = "github.com/lni/dragonboat/v3",
-        replace = "github.com/tylerwilliams/dragonboat/v3",
-        sum = "h1:rgT2w+T0OLmZoRSez1eiAwVgZ0ZIBE0zyzry++mUNbM=",
-        version = "v3.3.4-rc2",
+        sum = "h1:hh/XOeTC+fN1xJ5GsidYwKt2oeCe/V9WUjoB9dEKoJY=",
+        version = "v3.3.4",
```

Comment:
I think this will probably break my raft stuff. Do you know what made this change / how we can alias this package to my fork of it correctly?

Reply:
https://github.com/buildbuddy-io/buildbuddy/pull/1491 will fix the raft issue -- need a `replace` directive in `go.mod`. Gazelle respects the replace directive and will generate the correct `deps.bzl` rule with the `replace` attribute.

Reply:
Ack, thanks!